source/browseMode.py

```diff
@@ -1760,6 +1760,19 @@ def script_previousColumn(self, gesture):
 	# Translators: the description for the previous table column script on browseMode documents.
 	script_previousColumn.__doc__ = _("moves to the previous table column")
 
+	def script_toggleIncludeLayoutTables(self,gesture):
+		if config.conf["documentFormatting"]["includeLayoutTables"]:
+			# Translators: The message announced when toggling the include layout tables document formatting setting.
+			state = _("include layout tables off")
+			config.conf["documentFormatting"]["includeLayoutTables"]=False
+		else:
+			# Translators: The message announced when toggling the report tables document formatting setting.
+			state = _("include layout tables on")
+			config.conf["documentFormatting"]["includeLayoutTables"]=True
+		ui.message(state)
+	# Translators: Input help mode message for include layout tables command.
+	script_toggleIncludeLayoutTables.__doc__=_("Toggles on and off the inclusion of layout tables in browse mode")

```

Comment:
Do you need a category attribute here? I'm not familiar with this code.

Reply:
Nope, the category is set on the class itself, so all the scripts have the category that the class prescribes.